index.html

```diff
@@ -1,13 +1,67 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en-us">
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Zema's Portfolio</title>
   <link rel="stylesheet" href="styles.css">
-  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
-  <h1>Hello Microverse!</h1>
+  <header>
+    <div class="status-bars">
+
+    </div>
+    <div class="app-bar">
+      <div class="toolbar">
+        <nav>
+          <div class="normal-button">
+            <img class="burger" src="assets/burger.svg" alt="Menu icon.">
+          </div>
+        </nav>
+        <a href="#home">
+          <div class="logo">
+            <span>My Logo</span>
+          </div>
+        </a>
+        <div class="toolbar-container">
+          
+        </div>
+      </div>
+
+
+    </div>
+  </header>
+  <main>
+    <section class="headline">
+      <div class="primary-text-2">
+        <h1>I'm Zermarden <br>
+        Glad to see <br>
+        you!</h1>
+      </div>      
+      <div class="social-media-icons">
+        <ul class="social-media-icons-list">
+          <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="20" height="20"></a></li>
+          <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="20" height="20"></a></li>
```

Comment:
- Please, kindly ensure this links to your social media platform to help recruiters see your other professional social media presence. You add the ones you have and leave the ones you don't have yet as you'd update them later in the future.